Copy config on install

Jday7879 opened the pull request:
# Pull Request Title

Copy config and main script after installing package

# Summary

Minor changes from Chris' work on creating a script to copy over config and main script after installing package
Also corrects install issues

# Type of Change

<!--
Please select the type of change that applies to this pull request.
-->

- [x] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe):


# Checklists

<!--
These are do-confirm checklists; it confirms that you have done each item.

If actions are irrelevant, please add a comment stating why.

Incomplete pull/merge requests may be blocked until actions are resolved, or closed at
the reviewers' discretion.
-->

This pull request meets the following requirements:

## Creator Checklist

- [x] Installable with all dependencies recorded
- [x] Runs without error
- [x] Follows PEP8 and project-specific conventions
- [x] Appropriate use of comments, for example, no descriptive comments
- [x] Functions documented using Numpy style docstrings
- [x] Assumptions and decisions log considered and updated if appropriate
- [x] Unit tests have been updated to cover essential functionality for a reasonable range of inputs and conditions
- [x] Other forms of testing such as end-to-end and user-interface testing have been considered and updated as required

If you feel some of these conditions do not apply for this pull request, please
add a comment to explain why.

## Reviewer Checklist

- [ ] Test suite passes (locally as a minimum)
- [ ] Peer reviewed with review recorded

# Additional Information

Please provide any additional information or context that would help the reviewer understand the changes in this pull request.

# Related Issues

Link any related issues or pull requests here.

Jday7879 commented:
I have reviewed the original code from Chris' forked repo. Might need a glace over from someone else to ensure they are happy with my minor changes

AntonZogk reviewed (approved):
Amazing work @Jday7879 , works as expected both locally and CDP. 

Might be worth making mbs_results/utilities/copy_script_and_config.py executable and then adding a console_script argument in the package with a relevant command , something like

[options.entry_points]
console_scripts =
    setup= mbs_results.utilities.copy_script_and_config:copy_script_and_config
    
Thus the user can just type `setup` in command prompt , but this might be unnecessary complexity

Jday7879 commented:
Thanks @AntonZogk, Chris did all the hard work I just tweaked it! Added an optional entry point for the copy script and put an entry point for the main script while I was there. Tested both locally and worked, also updated the readme
